src/webapi/element.rs

```diff
@@ -42,6 +42,26 @@ pub trait IElement: IEventTarget {
             js!( return @{self.as_ref()}.querySelectorAll( @{selector} ); ).into_reference_unchecked().unwrap()
         }
     }
+    
+     /// A property which represents the inner html of a element and its descendants as a DOM string.
+    ///
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML)
+    fn inner_html( &self ) -> Option< String > {
```

Comment:
Why the `Option`? Can it return `null` or `undefined`? If it can't then it should simply be a `String`.